deploy/FlapperInit.sol

```diff
@@ -112,14 +112,14 @@ library FlapperInit {
 
         // Sanity checks
         require(flapper.spotter()  == address(dss.spotter),                       "Flapper spotter mismatch");
-        require(flapper.dai()      == cfg.dai,                                    "Flapper dai mismatch");
+        require(flapper.nst()      == cfg.nst,                                    "Flapper nst mismatch");
         require(flapper.pair()     == cfg.pair,                                   "Flapper pair mismatch");
         require(flapper.receiver() == dss.chainlog.getAddress("MCD_PAUSE_PROXY"), "Flapper receiver mismatch");
 
         PairLike pair = PairLike(flapper.pair());
-        (address pairDai, address pairGem) = pair.token0() == cfg.dai ? (pair.token0(), pair.token1())
+        (address pairDai, address pairGem) = pair.token0() == cfg.nst ? (pair.token0(), pair.token1())
```

Comment:
Some local vars, types and functions were not renamed:
- in `src`: `pairDai`, `reserveDai`, `DaiJoinLike` (in the Splitter).
- in `test`: `reserveDai`, `minimalDaiReserve`, `currentDaiForGem`, `UNIV2_LINK_DAI_PAIR`, `_getDaiToSell`, `uniV2GemForDai`, `uniV2DaiForGem`, and some test function names. 

Was that intentional to reduce the diff? I'd tend to at least fully do the renaming for the files in `src`.